functions/processQueueItem.js

```diff
@@ -79,12 +89,25 @@ module.exports.handler = async function({ ReceiptHandle, Body }) {
         positive: upstreamServiceResponse.IsMatch
       }
     });
+    const timingSubmitted = Date.now() - timingSubmittedStart;
 
     const { QueueUrl } = await SQS.getQueueUrl({
       QueueName: QUEUE_NAME
     }).promise();
 
     await SQS.deleteMessage({ QueueUrl, ReceiptHandle }).promise();
+
+    await Metrics.workerWorks({
+      consumer_name: user,
+      worker_id: awsRequestId,
+      watchdog_id: id,
+      photodna_tracking_id: upstreamServiceResponse.TrackingId,
+      is_error: false,
+      is_match: upstreamServiceResponse.IsMatch,
+      timing_sent: timingSent,
+      timing_received: timingReceived,
+      timing_submitted: timingSubmitted
```

Comment:
The timing values don't match what's in the metrics doc.

- It looks like `timing_sent` is meant to be `timing_retrieved`.
- `timing_retrieved` is missing.
- `timing_received` appears to be `timing_sent` and `timing_received` combined. (Although I don't think we can break that request timing down?)